### ISSUE TITLE
pandoc.exe: Out of memory  Error: pandoc document conversion failed with error 251

### ISSUE THREAD

**Issue by josegabrielbcn** [rstudio/rmarkdown#1535]
@yihui Did you find any way to solve this issue? Out of memory Error: pandoc document conversion failed with error 251
@AlbertoBernat

**Comment by cderv** [comment]
I think this issue is very dependant of your rmarkdown file. This is an issue with pandoc and how it handles memory. Probably related to [this pandoc issue](https://github.com/jgm/pandoc/issues/3169) and there is a lot of occurrence when you search for this issue.
There is also knitr one showing the specificity of this type of issue. https://github.com/yihui/knitr/issues/1152
There seems to be different fixing all consisting of reducing the memory consumption during pandoc conversion. 


Can you share a reproducible example or more details on your specific case ? 

I don't think this is a rmarkdown issue. You could also find some help on community.rstudio.com

**Comment by mlduarte** [comment]
FYI.  in my case I was inadvertently printing out a very long data frame using `kable()` (as-is, no scrolling, etc - not sure if would have helped but regardless I didn't want to include it in the output .html).    I was able to generate the .html manually using `rmarkdown::render('myfile.Rmd')` despite the knit button giving an error.  If this didn't work, I was going to render parts of the document, stopping later and later until I found the cause of the out of memory error.  Someone has suggested: `knitr::knit_exit()`
